docs/guide/web-simulation.md

```diff
@@ -0,0 +1,259 @@
+    deactivate W
+    U->>C: Start simulation
+    Note left of C: webots.min.js
+    C->>SE: Open simulation session
+    activate SE
+      SE-->>C: Return simulation server web socket URL
+    deactivate SE
+    C->>SS1: Start simulation
+    activate SS1
+      SS1->>W: Download simulation project
+      activate W
+        Note right of W: download_project.php
+        W-->>SS1: Return simulation project archive
+      deactivate W
+      SS1->>SW1: Start Webots
+      activate SW1
+        SS1-->>C: Return Webots web socket URL
+      deactivate SS1
+      C->>SW1: Register client
+      SW1->>C: Send simulation worlds status
+      U->>C: Run simulation
+      C->>SW1: Change simulation running mode
+      loop Each simulation step
+        SW1->>C: Send simulation world status
+      end
+      U->>C: Edit and save controller
+      C->>W: Send new file
+      activate W
+        Note right of W: upload_file.php
+        W-->>C: Return OK
+      deactivate W
+      C->>SW1: Send new file
+      U->>C: Reset simulation
+      C->>SW1: Reset simulation
+      loop Each simulation step
+        SW1->>C: Send simulation worlds status
+      end
+      C-->>SS1: {Client disconnected}
+    deactivate C
+    activate SS1
+      SS1->>SW1: Kill Webots
+    deactivate SW1
+  deactivate SS1
+%end
+%end
+
+#### Session Server
+
+The session server is the entry point for creating a Webots instance and starting the simulation.
+It manages the load of the simulation server machines and sends the URL of the available simulation server to the client.
+
+These are the configuration parameters for the session server:
+- `port`: local port on which the server is listening.
+- `server`: host where this session script is running.
+- `administrator`: email address of administrator that will receive notifications about the status of the simulation server machines.
+- `mailServer`: mail server host from which the notifications are sent.
+- `mailSender`: email address used to send the notifications.
+- `simulationServers`: lists all the available simulation servers.
+- `sslCertificate`: path to the certificate file for a SSL enabled server.
+- `sslKey`: path to the private key file for a SSL enabled server.
+- `logDir`: directory where the log file is written. Default value is "WEBOTS\_HOME/resources/web/server/log/session".
+
+Sending a AJAX request to the session server URL will query the availability of the simulation servers and return 1 if some are available or 0 if no simulation server are available.
+
+The `/session` request checks for available simulation servers and returns the URL of the minimally loaded server.
+
+The `/monitor` request opens a page showing an overview of the available simulation servers and their load.
+
+#### Simulation Server
+
+The simulation server creates and starts a Webots instance with the desired simulation for each client request and sends the WebSocket URL of Webots to the client so that it can communicate directly.
```

Comment:
```suggestion
The simulation server creates and starts a Webots instance with the desired simulation for each client request and sends the WebSocket URL of Webots to the client so that it can communicate directly with Webots.
```